my-app/src/css/display.css

```diff
@@ -15,68 +17,62 @@ body, html {
 }
 
 .rowContainer{
-  width: 1950px;
-  overflow-x: auto;
-  white-space: nowrap;
+  width: 100%;
+  /* white-space: normal; */
 }
+
 .row {
   margin-top: 50px;
   display: flex;
   padding: 10px;
   justify-content: space-between;
+  margin: 0 auto;
+  max-width: 90%;
+  overflow-y: auto;
 }
 
 .button {
-  width: 330px;
-  height: 186px;
-  display: flex;
+  width: 445px;
+  height: 246px;
   flex-direction: column;
   align-items: center;
-  border: 2px solid #000000;
   filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
   border-radius: 20px;
-  margin: auto
+  margin-right: 20px;
 }
 
 .button img {
-  width: 200px;
-  height: 186px;
-  background-size: cover;
+  width: 444px;
+  height: 240px;
+  border-radius: 20px;
+  object-fit: cover;
 }
 
 .button span {
   font-family: 'Julius Sans One';
-  font-size: 14px;
+  font-size: 45px;
   text-align: center;
-}
-
-.WhatToEat{
-  font-family: 'Julius Sans One';
-  width: 237px;
-  height: 108px;
-  left: 59px;
-  top: 83px;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 50px;
-  color: #000000;
-  text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  margin-top: 25px;
-  margin: 55px;
+  align-items: center;
```

Comment:
This actually goes with Button Span but git is showing its apart of WhatToEat which has been moved